kie-osgi/kie-karaf-features/src/main/filtered-resources/repository/features-core.xml

```diff
@@ -30,7 +30,7 @@
 
   <feature name="drools-jpa" version="${project.version}" description="Drools JPA">
     <feature version="${project.version}">drools-module</feature>
-    <feature version="[4,5)">spring-orm</feature>
+    <feature version="[4,6)">spring-orm</feature>
```

Comment:
I have a question about minimal version. Does it make sense to keep it on 4 if we want to update to Spring 5?

Reply:
If you don't plan supporting Spring 4, then sure, it can be `[5,6)`. I just wanted to change as little as possible.

Reply:
Understand. I agree to keep it as it is and leave this to another PRs. It depends on how will discussions about supported versions finish.